shared/FFTtrials.m

```diff
@@ -1,4 +1,4 @@
-function [FFT] = FFTtrials(cfg, force)
+function [FFT] = FFTtrials(cfg, LFP, force)
```

Comment:
I don't see how this will not break when LFP is not given, as it is used now. If you want to add this extra field, make sure to deal with the previous use case.